amd_openvx_extensions/amd_rpp/source/internal_publishKernels.cpp

```diff
@@ -123,6 +123,7 @@ vx_status get_kernels_to_publish()
     STATUS_ERROR_CHECK(ADD_KERENEL(ResizeCropMirrorPD_Register));
     STATUS_ERROR_CHECK(ADD_KERENEL(Copy_Register));
     STATUS_ERROR_CHECK(ADD_KERENEL(Nop_Register));
+    STATUS_ERROR_CHECK(ADD_KERENEL(ResizeMirrorNormalizeTensor_Register));
```

Comment:
@SundarRajan28 Need to do a code clean up and correct the spelling for "kernel" everywhere

Reply:
All the kernel names are changed accordingly for fixing the spelling mistake